packages/components/src/editable-page-header/index.js

```diff
@@ -29,24 +30,39 @@ const EditablePageHeader = ( { onChange, text, disabled } ) => {
 	};
 	const hideForm = () => setActive( false );
 
+	const [ showButton, setShowButton ] = useState( false );
+
 	const classes = classnames( 'editable-page-header', {
 		'is-disabled': disabled,
 	} );
 
 	return (
-		<div className={ classes }>
-			{ ! active && (
-				<PageHeader onClick={ showForm }>{ text }</PageHeader>
-			) }
-
-			{ active && (
-				<Form
-					value={ text }
-					onChange={ onChange }
-					onCancel={ hideForm }
-				/>
-			) }
-		</div>
+		<>
+			<div
+				className={ classes }
+				onMouseEnter={ () => setShowButton( true ) }
+				onMouseLeave={ () => setShowButton( false ) }
+			>
+				{ ! active && (
+					<PageHeader onClick={ showForm }>{ text }</PageHeader>
+				) }
+				{ ! active && showButton && (
+					<Button
+						className="editable-page-header__button"
+						onClick={ showForm }
+					>
+						__( { ( 'Edit', 'dashboard' ) } )
```

Comment:
Something is off with this line :D 
<img width="355" alt="image" src="https://user-images.githubusercontent.com/157240/166313076-fb530a75-d068-4542-bcbf-e03dfa42710e.png">